apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -614,6 +617,26 @@ init_per_testcase(TC, Config) when TC == sophia_auth_tx;
         true  -> init_per_testcase_common(TC, Config);
         false -> {skip, {requires_protocol, iris, TC}}
     end;
+init_per_testcase(TC = bad_aens_pointer_handling_lima_to_iris, Config) ->
+    case aect_test_utils:latest_protocol_version() of
+        ?IRIS_PROTOCOL_VSN ->
+            LHeight = 20,
+            IHeight = 25,
+            Fun = fun(H) when H <  IHeight -> ?LIMA_PROTOCOL_VSN;
```

Comment:
Perhaps it doesn't matter, but shouldn't it be `H >= LHeight, H < IHeight`?

Reply:
Yeah, the particular Lima height is not important.